pages/patterns/create-a-profile/pronouns.md

```diff
@@ -112,7 +112,6 @@ Provide a text entry field that supports a rich array of special characters and
 {:.site-component-section-title}
 ## References
 - Ask users for gender or sex. (n.d.) Retrieved on July 19, 2022, from [https://design-system.service.gov.uk/patterns/gender-or-sex/](https://design-system.service.gov.uk/patterns/gender-or-sex/)
-- Asking about gender in online forms. (September 18, 2015) Retrieved on July 19, 2022, from [http://www.practicemakesprogress.org/blog/2015/9/18/asking-about-gender-on-online-forms](http://www.practicemakesprogress.org/blog/2015/9/18/asking-about-gender-on-online-forms)
```

Comment:
This is returning a 404 and causing an html proofer error. It appears as though the website is broken and I could not find a replacement link.

Reply:
Biar aja saya miskin , soal nya saya gak curang. Emank kenapa kalo saya miskin , emnk kamu bisa memperkaya saya , harta mu cukup untuk beli beras , gak cukup untuk bayar negara haha